From daf17dae1e233d5fdf1915073b7f163355a16c15 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Tue, 21 Jan 2014 10:44:12 +0800
Subject: [PATCH 0572/5242] MLK-11703-7: ARM: dts: imx6qdl-sabresd: add
 retain-state-suspended property in dts

commit  d3168310f564e9a96ffaee461155c8846fb034cd from
https://source.codeaurora.org/external/imx/linux-imx.git

cherry-pick below patch:

ENGR00295892-2: ARM: dts: imx6qdl-sabresd: add retain-state-suspended property in dts

Add property "retain-state-suspended" in dts.

Signed-off-by: Robin Gong <b38343@freescale.com>
(cherry picked from commit aff480769c26ff7f64059f9631b10e8177c4791b)
(cherry picked from commit 4a029915c4693b16db4806f078504cc86f34c069)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index 8ed9fe5..430c6ed 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -48,6 +48,7 @@
 		charger-led {
 			gpios = <&gpio1 2 0>;
 			linux,default-trigger = "max8903-charger-charging";
+			retain-state-suspended;
 		};
 	};
 
-- 
1.7.9.5

